lightning/src/ln/channel.rs

```diff
@@ -4204,6 +4204,7 @@ impl<Signer: Sign> Channel<Signer> {
 				Some(script) => script.clone().into_inner(),
 				None => Builder::new().into_script(),
 			}),
+			channel_type: None,
```

Comment:
unrelated to this diff, but if the message is to obtain an unprompted message to send, "generate" is probably a better prefix than "get"

Reply:
Yea, :man_shrugging:, I mean its getting *the* open channel? I'm not gonna bother changing it unless you feel strongly.

Reply:
Fair point